drv/lpc55-update-server/src/main.rs

```diff
@@ -176,6 +183,7 @@ impl idl::InOrderUpdateImpl for ServerImpl {
         }
 
         do_block_write(
+            &mut self.flash,
```

Comment:
I can't put this comment at the correct line, but the final argument to this function could be
```rust
            self.header_block.as_ref().unwrap_lite(),
```
(no need for `as_mut`)

Reply:
Changed!